js/index.js

```diff
@@ -1354,8 +1357,10 @@ function DataManager() {
 			});
 		} else {
 			oTable.fnClearTable();
-			oTable.fnAddData(dataList);
-			oTable.fnAdjustColumnSizing();
+			if (dataList.length > 0){
+				oTable.fnAddData(dataList);
+				oTable.fnAdjustColumnSizing();
+			}
```

Comment:
Is it possible to have a dump of that DB and the URL where the problem appears (if this if is not done)? `full` should be the object, it makes no sense that it's can with an empty array.